### ISSUE TITLE
Installer fails when prompting for secrets

### ISSUE THREAD

**Issue by jwhonce** [openshift/installer#915]
# Version

```console
$ openshift-install version
bin/openshift-install v0.5.0-master-116-g4f1e3967a490578c1fb3369b3c6dfb2d1165a853$ 
```

# Platform (aws|libvirt|openstack):

AWS

# What happened?

When allowing openshift-installer to prompt for the AWS credentials, it failed after  asking for access key.

# What you expected to happen?

Accept the access key and prompt for the secret key, then finish the install.

# How to reproduce it (as minimally and precisely as possible)?

Do not run ```aws configure```.

```console
$ bin\openshift-installer cluster up
```

# Anything else we need to know?

AWS credentials are original form V1 and V2

# References

**Comment by staebler** [comment]
Fixed in 4a497123063d8230f9100a1fa41b2cfbd68c5797.

**Comment by abhinavdahiya** [comment]
> Fixed in [4a49712](https://github.com/openshift/installer/commit/4a497123063d8230f9100a1fa41b2cfbd68c5797).

/close
